samples/zephyr/subsys/mgmt/mcumgr/smp_svr/sysbuild/mcuboot/boards/nrf54l15dk_nrf54l10_cpuapp_ext_flash.overlay

```diff
@@ -1,5 +1,6 @@
 / {
 	chosen {
+		zephyr,code-partition = &boot_partition;
```

Comment:
Why is this necessary? We only support nRF54L with partition manager, so `zephyr,code-partition` should not matter - unless there are some strange CMake checks I am not aware of